src/elexmodel/handlers/data/Estimandizer.py

```diff
@@ -0,0 +1,153 @@
+import numpy as np
+
+
+class Estimandizer:
+    """
+    Estimandizer. Generate estimands explicitly.
+    """
+
+    def __init__(self, data_handler, estimands):
+        self.data_handler = data_handler
+        self.estimands = estimands
+        self.transformations = []
+        self.transformation_map = {
+            "margin": [self.calculate_margin],
+            "voter_turnout_rate": [self.calculate_voter_turnout_rate],
+            "standardized_income": [self.standardize_median_household_income],
+            "age_groups": [self.create_age_groups],
+            "party_vote_share": [self.calculate_party_vote_share],
+            "education_impact": [self.calculate_party_vote_share, self.analyze_education_impact],
+            "gender_turnout_disparity": [self.investigate_gender_turnout_disparity],
+            "ethnicity_voting_patterns": [self.calculate_party_vote_share, self.examine_ethnicity_voting_patterns],
+            "income_impact": [
+                self.calculate_party_vote_share,
+                self.standardize_median_household_income,
+                self.explore_income_impact,
+            ],
+            "candidate": [self.candidate],
+        }
+
+    def check_estimand(self, estimand):
```

Comment:
I think we want to check the data and not have a pre-specified list here

Reply:
On the Jira ticket it says that "since dem_votes, gop_votes and total_votes are already included in the data, this estimand generator should do nothing (or over-write those fields)" so that's why I had a pre-specified list. Should I switch this so that we have a parameter of estimands that we want to check, inclusive of these three?

Reply:
My thinking was that we could check the data and see if the name of the estimand, i.e. `dem_votes`, exists in the data and if it does, do not create it. Does that make sense?

Reply:
I added a function at the start of generate_estimands that creates dem_votes, gop_votes and total_votes in a G election if they do not exist. If the data handler is pre-processed and already has the result data (results_dem, results_gop, results_total), it will use that. Otherwise, if the data handler is combined, I have it set such that it creates empty columns with these names. Is this what you mean?